src/logplex_app.erl

```diff
@@ -117,6 +117,10 @@ cache_os_envvars() ->
                         integer}
                       ,{api_endpoint_url, ["LOGPLEX_API_ENDPOINT_URL"],
                        optional}
+                      ,{tls_cacertfile, ["LOGPLEX_TLS_CACERTFILE"],
+                        optional}
```

Comment:
the OS environment variable is. If that is not set then the default specified in the .app file will take over.